client/components/data/query-viewers/README.md

```diff
@@ -0,0 +1,46 @@
+# QueryViewers
+
+`<QueryViewers />` is a React component used in managing network requests for viewers of a given site.
+
+## Usage
+
+Render the component, passing `siteId`, `page` and `numbers`. It does not accept any children, nor does it render any elements to the page. You can use it adjacent to other sibling components which make use of the fetched data made available through the global application state.
+
+```jsx
+import React from 'react';
+import { useSelector } from 'react-redux';
+import QueryViewers from 'calypso/components/data/query-viewers';
+import { getViewers } from 'calypso/state/selectors/get-viewers';
+
+export default function ViewersList( { siteId } ) {
+	const siteId = 1;
+	const viewers = useSelector( ( state ) => getViewers( state, siteId ) );
+
+	return (
+		<div>
+			<QueryViewers siteId={ siteId } page={ 2 } />
+
+			{ viewers?.map( ( viewer ) => (
+				<div key={ viewer.ID }>{ viewer.name }</div>
+			) ) }
+		</div>
+	);
+}
+```
+
+## Props
+
+### `siteId`
+
+<table>
+	<tr><th>Type</th><td>Number</td></tr>
+	<tr><th>Required</th><td>Yes</td></tr>
+</table>
+
+### `page`
+
+<table>
+	<tr><th>Type</th><td>Number</td></tr>
+	<tr><th>Required</th><td>No</td></tr>
+	<tr><th>Default</th><td>`1`</td></tr>
```

Comment:
I've noticed we're documenting `page` as defaulting to `1`, but didn't see the corresponding change in the code. Could you point me as to where we're defaulting to `1`?

Reply:
fixed in https://github.com/Automattic/wp-calypso/pull/48137/commits/58370b3cd47d6e7fe5bfe12d292e5ced417587ab